### PR TITLE
Prime Numbers upto a user-desired value.

### PR DESCRIPTION
**Pull request by AdharshM**
Fixes #

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- 
- 

#### Languages Used:
- C++14
- 

#### Files Added:
- 
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Sieve_Of_Eratosthenes/Sieve_Of_Eratosthenes.cpp [hunk 1]
```diff
@@ -29,7 +29,8 @@ void SieveOfEratosthenes(int n)
 // Program to test above function
 int main()
 {
-    int n = 30;
+    int n ;
```
Comment (MastersAbh) [inline Sieve_Of_Eratosthenes/Sieve_Of_Eratosthenes.cpp]: dont give space after n

**Comment by MastersAbh** [comment]
Also, please mention which issue # you are fixing

**Comment by MastersAbh** [comment]
@AdharshM  Please mention the issue # or close this PR if it is not assigned to any issue

**Comment by jainaman224** [comment]
@AdharshM Please fill the PR template properly.
Are you working on it?

**Pull request closed without merging**